docs/docs/tutorial/index.md

```diff
@@ -33,11 +32,14 @@ After ensuring the tools above are available, begin by downloading pre-built bin
 ```shell
 mkdir -p bin
 version=0.17.0
+mithril_version=2423.0
+node_version=8.9.3
```

Comment:
Why node 8.9?

Reply:
@ch1bo suggested this was the right version

i was trying 8.11, which is what is was _in the docs here_ ( but _not_ in the published docs, where it's also 8.9 - https://hydra.family/head-protocol/docs/tutorial/ ) but that release doesn't even exist on the node repo; i.e. this url - https://github.com/IntersectMBO/cardano-node/releases/download/8.11.0/cardano-node-8.11.0-linux.tar.gz - doesn't even exist.

Reply:
8.9.3 works with this mithril release and seemingly also with the hydra-node.

I'd say we use 8.9.3 for now here, but also bump it here when we switch to 9.0